docs/source/keras.rst

```diff
@@ -0,0 +1,45 @@
+Keras
```

Comment:
Perahaps title this Keras and Tensorflow, to make it easier for people just familiar with tensorflow to find this?